zulipterminal/ui_tools/views.py

```diff
@@ -925,3 +928,19 @@ def __init__(self, controller: Any, msg: Message) -> None:
              for index, (field, data) in enumerate(msg_info.items())]
 
         super().__init__(controller, msg_info_content, 'MSG_INFO')
+
+
+class EditHistoryView(PopUpView):
+    def __init__(self, controller: Any, msg: Any) -> None:
+        self.controller = controller
+        self.msg = msg
+        self.width = 64
+        self.height = 0
+        super().__init__(controller, [], 'MSG_INFO')
```

Comment:
The `MSG_INFO` makes me wonder if it'd be a better idea to rename the `self.command` variable to something like `self.exit_popup_command`.